packages/flutter_tools/gradle/flutter.gradle

```diff
@@ -828,6 +835,9 @@ abstract class BaseFlutterTask extends DefaultTask {
             if (extraGenSnapshotOptions != null) {
                 args "--ExtraGenSnapshotOptions=${extraGenSnapshotOptions}"
             }
+            if (treeShakeIcons == true) {
```

Comment:
nit: the ExtraGenSnapshotOptions is a bit weird, so place the `-d` above it so it aligned within the other define options

Reply:
Done